test/auth/features/oidc/sequence.go

```diff
@@ -54,7 +56,7 @@ func SequenceSendsEventWithOIDC() *feature.FeatureSet {
 		Name: "Sequence send events with OIDC support",
 		Features: []*feature.Feature{
 			SequenceSendsEventWithOIDCTokenToSteps(),
-			SequenceSendsEventWithOIDCTokenToReply(),
+			//SequenceSendsEventWithOIDCTokenToReply(),
```

Comment:
```suggestion
			SequenceSendsEventWithOIDCTokenToReply(),
```

Reply:
i was in testing it by commenting out the reply portion. I will uncomment it